carapace-server/src/main/java/org/carapaceproxy/core/Listeners.java

```diff
@@ -148,6 +149,7 @@ private void stopListener(HostPort hostport) throws InterruptedException {
         ListeningChannel channel = listeningChannels.remove(hostport);
         if (channel != null) {
             channel.channel.disposeNow(Duration.ofSeconds(10));
+            FutureMono.from(channel.getConfig().getGroup().close()).block(Duration.ofSeconds(30));
```

Comment:
I'm not sure if 30s is too much here.

Maybe we can go on with this, and change it or make it configurable if it becomes a problem

Reply:
Maybe we can reduce to 10s?

Reply:
It might be an improvement, for sure